docs/07-AdvancedUsage.md

```diff
@@ -217,6 +217,28 @@ These examples can be written using Doctrine-style annotation syntax as well:
   }
 ```
 
+You can also use the `@dataprovider` annotation for creating dynamic examples, using a public static method for providing example data:
```

Comment:
why *static* method? To be compatible with PHPUnti it should be an instance method. And I don't see any technical limitation why not.